engines/pyfunc-ensembler-job/Dockerfile

```diff
@@ -86,7 +86,7 @@ ENV PROJECT_DIR $HOME/batch-ensembler
 RUN mkdir -p $PROJECT_DIR
 COPY --chown=$UID:$GID . $PROJECT_DIR/
 # Hack to ensure that it's compatible with the ../../sdk stated in requirements.txt
-COPY ./temp-deps/sdk $PROJECT_DIR/../../sdk
+COPY --chown=$UID:$GID ./temp-deps/sdk $PROJECT_DIR/../../sdk
```

Comment:
Had to introduce this due to permission errors installing the `turing-sdk` python package from a local directory: https://github.com/deadlycoconuts/turing/actions/runs/3572688644/jobs/6006007941#step:4:993

Apparently the error's linked to recent `pip` updates. When Python 3.10 is used, we are using `pip=22.2.2` instead of `pip=21.2.2`, when other Python versions are used. In version `21.3` of `pip` however, a [change](https://github.com/pypa/pip/pull/10495) was made to make in-tree builds the default for installing from local directories. While that change was meant to address this [issue](https://github.com/pypa/pip/issues/7555), the change was ultimately a reintroduction of a change that had originally been introduced before being subsequently [reverted](https://github.com/pypa/pip/issues/8196).

During the time when this change was initially made, a user of a certain random other repo has encountered the same issue while building from a (copied) local directory in a Docker image and his fix can be found here, https://github.com/stellargraph/stellargraph/pull/1407.